Library/include/CSP/Systems/ECommerce/ECommerce.h

```diff
@@ -0,0 +1,155 @@
+/*
+* Copyright 2023 Magnopus LLC
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+#include "CSP/CSPCommon.h"
+#include "CSP/Common/Map.h"
+#include "CSP/Common/String.h"
+#include "CSP/Services/WebService.h"
+
+namespace csp::services
+{
+
+CSP_START_IGNORE
+template <typename T, typename U, typename V, typename W> class ApiResponseHandler;
+CSP_END_IGNORE
+
+} // namespace csp::services
+
+
+
+namespace csp::systems
+{
+/// @ingroup ECommerce System
+/// @brief Represents currency information for a product
+class CSP_API CurrencyInfo
+{
+public:
+	CurrencyInfo() = default;
+
+	/// @brief Currency Amount.
+	double Amount;
+	/// @brief Currency type
+	csp::common::String CurrencyCode;
+};
+
+/// @ingroup ECommerce System
+/// @brief Represents media information for a product
+class CSP_API ProductMediaInfo
+{
+public:
+	ProductMediaInfo() = default;
+
+	/// @brief Type of media content used.
+	csp::common::String MediaContentType;
+	/// @brief Alternative description of the media.
+	csp::common::String Alt;
+	/// @brief Url of the media.
+	csp::common::String Url;
+	/// @brief Width of the media.
+	int32_t Width;
+	/// @brief Height of the media.
+	int32_t Height;
+};
+
+/// @ingroup ECommerce System
+/// @brief Represents option for a variant
+class CSP_API VariantOptionInfo
+{
+public:
+	VariantOptionInfo() = default;
+
+	/// @brief Id of the variant option.
+	csp::common::String Name;
+	/// @brief Value of variant option.
+	csp::common::String Value;
+};
+
+/// @ingroup ECommerce System
+/// @brief Represents variant information for a product
+class CSP_API ProductVariantInfo
+{
+public:
+	ProductVariantInfo() = default;
+	/// @brief Id of the variant.
+	csp::common::String Id;
+	/// @brief Title of the variant.
+	csp::common::String Title;
+	/// @brief Title of variant.
```

Comment:
You have a duplicate of 'Title of the variant'

Reply:
sorted